sdk/storage/azblob/models.go

```diff
@@ -27,7 +27,7 @@ func (iter *ListContainersIterator) NextPage(ctx context.Context) (*ListContaine
 	if err != nil {
 		return nil, err
 	}
-	page, err := iter.client.s.ListContainersCreateResponse(resp)
+	page, err := iter.client.s.ListContainersHandleResponse(resp)
```

Comment:
Yes, all looks good but remember we need a CurrentPageMarker "property" on the iterator too.

Reply:
It's already in the returned page, you want to add it to the iterator itself?